tests/payment/models/Payment - Credit card - Expired.model.js

```diff
@@ -0,0 +1,64 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getEmailWithSomeExpiredCard,
+  confirmSecondPurchase,
+  getEmailWithAllExpiredCards,
+} from '../../../utils/profile-actions'
+import {
+  selectCreditCardGroup,
+  queryIframe,
+  getIframeBody,
+} from '../../../utils/payment-actions'
+
+export default function test(account) {
+  describe(`Payment - Credit Card  - Expired - ${account}`, () => {
+    beforeEach(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with two valid and one expired cards', () => {
+      const email = getEmailWithSomeExpiredCard()
+
+      setup({ skus: ['289'], account })
+      fillEmail(email)
+      confirmSecondPurchase()
+      selectCreditCardGroup()
+      queryIframe($iframe => {
+        const body = getIframeBody($iframe)
+
+        cy.wrap(body)
+          .contains('Cartão vencido')
+          .should('be.visible')
+        cy.wrap(body)
+          .find('.ExpiredSavedCard')
+          .should('exist')
+        cy.wrap(body)
+          .find('.ExpiredSavedCard')
+          .should('not.have.class', 'active')
```

Comment:
can we check for the disabled attribute here?
```suggestion
          .should('be.disabled')
```